pkg/exchange/bybit/convert.go

```diff
@@ -368,7 +368,7 @@ func toLocalInterval(interval types.Interval) (string, error) {
 func toGlobalKLines(symbol string, interval types.Interval, klines []bybitapi.KLine) []types.KLine {
 	gKLines := make([]types.KLine, len(klines))
 	for i, kline := range klines {
-		endTime := types.Time(kline.StartTime.Time().Add(interval.Duration()))
+		endTime := types.Time(kline.StartTime.Time().Add(interval.Duration() - time.Millisecond))
```

Comment:
Could you comment on why using minus 1ms?

Reply:
Updated. And i also update the types.KLine.EndTime comment

```
// EndTime follows the binance rule, to avoid endTime overlapping with the next startTime. So if your end time (2023-01-01 01:00:00)
	// are overlapping with next start time interval (2023-01-01 01:00:00), you should subtract -1 time.millisecond on EndTime.
```